src/affine.jl

```diff
@@ -17,7 +17,14 @@ struct Affine{N,T<:AbstractFloat}
 end
 
 interval(x::Affine) = x.range
+range(x::Affine) = x.range
```

Comment:
The reason I stopped using `range` is because `range` now means something very different in Base.

Reply:
Ok, i opened https://github.com/JuliaIntervals/AffineArithmetic.jl/issues/17.